torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -2083,25 +2083,41 @@ std::tuple<at::Tensor, at::Tensor> AtenXlaType::nll_loss_forward(
 at::Tensor AtenXlaType::norm(const at::Tensor& self,
                              c10::optional<at::Scalar> p,
                              at::ScalarType dtype) {
+  // p = 0 is nonzero, which is not lowered to XLA
```

Comment:
Maybe: `If p==0 it is a torch.nonzero(), which is not lowered to XLA due to dynamic shapes issue.`
Same below.